release/nightly_tests/stress_tests/stress_tests_app_config.yaml

```diff
@@ -5,7 +5,7 @@ python:
   pip_packages:
     - terminado
     - boto3
-    - cython==0.29.0
+    - cython==0.29.26
```

Comment:
FYI @rkooo567 @simon-mo  is weird we have to have this in the stress test config.

Reply:
+1, let's try to remove it in a follow up PR :)